flow/Trace.h

```diff
@@ -384,32 +384,21 @@ struct SpecialTraceMetricType
 
 TRACE_METRIC_TYPE(double, double);
 
-struct TraceEvent {
-	TraceEvent();
-	TraceEvent(const char* type, UID id = UID()); // Assumes SevInfo severity
-	TraceEvent(Severity, const char* type, UID id = UID());
-	TraceEvent(struct TraceInterval&, UID id = UID());
-	TraceEvent(Severity severity, struct TraceInterval& interval, UID id = UID());
+struct BaseTraceEvent {
```

Comment:
Please explain in a comment that the difference between TraceEvent and BaseTraceEvent is that you can't call suppression-related calls on BaseTraceEvent but you can on TraceEvent